airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/job_factory/OAuthConfigSupplier.java

```diff
@@ -69,15 +74,32 @@ public JsonNode injectDestinationOAuthParameters(UUID destinationDefinitionId, U
     }
   }
 
-  private void injectJsonNode(ObjectNode config, ObjectNode fromConfig) {
+  @VisibleForTesting
+  void injectJsonNode(ObjectNode mainConfig, ObjectNode fromConfig) {
```

Comment:
it was cumbersome to test this via the calling methods so I exposed it here. Open to changing this if we think it's a bad idea.